### PR TITLE
Increase disk size and use high CPU machine type in cloud build

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/813)

### CODE REVIEW

File: components/third_party_licenses.csv [hunk 1]
```diff
@@ -180,3 +180,4 @@ webencodings,https://raw.githubusercontent.com/gsnedders/python-webencodings/mas
 google-api-core,https://raw.githubusercontent.com/googleapis/google-cloud-python/master/LICENSE,Apache 2.0
 google-resumable-media,https://raw.githubusercontent.com/googleapis/google-resumable-media-python/master/LICENSE,Apache 2.0
 trainer,https://raw.githubusercontent.com/kubeflow/pipelines/master/LICENSE,Apache 2.0
+pyarrow,https://raw.githubusercontent.com/apache/arrow/master/LICENSE.txt,Apache 2.0
```
Comment (IronPan) [inline components/third_party_licenses.csv]: is this still needed for this PR?

Comment (hongye-sun) [reply components/third_party_licenses.csv]: I though it was added in my last PR. Not sure why it's not merged in master.

Comment (hongye-sun) [reply components/third_party_licenses.csv]: I dropped the change in the PR.

**Pull request merged**